templates/user/favorites.html

```diff
@@ -4,12 +4,13 @@
 
 {% block content %}
 
-<div class="container-md text-center">
-    <div class="h2 text-center">Your favorites</div>
-    <div class="row">
+<div class="container-md text-center mb-3">
+    <div class="h2 text-info text-center">Your favorites</div>
+    <div class="results row justify-content-center">
         {% if favorites %}
         {% for recipe in favorites %}
-        <div class="card text-center col-12 col-sm-6 col-md-5 col-lg-4 col-xl-3" style="width: 18rem;">
+        <div class="card text-center col-10 col-xs-10 offset-xs-1 col-sm-5 offset-sm-0 col-md-4 col-lg-3"
+            style="width: 18rem;">
```

Comment:
Inline style alert - try to keep the styles all in one place, unless there's a good reason not to do it.

Reply:
Yeah, that style isn't even doing anything. It was leftover from earlier stuff. Thanks for the catch. 😅